build.gradle

```diff
@@ -28,9 +28,29 @@ dependencies {
 
     testImplementation 'org.apache.httpcomponents:httpclient:4.5.13'
     testImplementation 'com.github.tomakehurst:wiremock-jre8:2.27.2'
-}
+    // https://mvnrepository.com/artifact/io.springfox/springfox-swagger2
+    implementation 'io.springfox:springfox-swagger2:3.0.0'
+// https://mvnrepository.com/artifact/io.springfox/springfox-swagger-ui
+    implementation 'io.springfox:springfox-swagger-ui:3.0.0'
 
+}
+test {
+    finalizedBy jacocoTestReport
+}
 
 tasks.named('test') {
     useJUnitPlatform()
 }
+
+jacoco {
+    toolVersion = "0.8.7"
```

Comment:
Why didn't you use the latest version of Jacoco?

Reply:
:(